packages/client/bin/cli.ts

```diff
@@ -307,7 +341,7 @@ async function run() {
     debugCode: args.debugCode,
     discDns: args.discDns,
     discV4: args.discV4,
-    mine: args.mine,
+    mine: args.dev ?? args.mine,
```

Comment:
hm with the nullish coalescing operator if `args.dev` is false it would return `false`:

> The nullish coalescing operator (??) is a logical operator that returns its right-hand side operand when its left-hand side operand is `null` or `undefined`, and otherwise returns its left-hand side operand.

```shell
➜  ~ ts-node
> false ?? true
false
```

Reply:
Ah, thanks for the clarification.  I actually need something slightly different `mine: args.mine || args.dev` to make this work.  We want to start the miner if either `args.mine===true` or if `args.dev===true`